### DISCUSSION TITLE
tool/baidu/callback

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#146] [Announcements]
# tool/baidu/callback

import Callback from 

https://alist.nn.ci/tool/baidu/callback.html

**Comment by tonylinzhen** [comment]
ErrorErrorError

**Comment by 3028356499** [comment]
不行啊一直转圈

**Comment by JekyKe** [comment]
我正常獲取了

**Comment by gch0509** [comment]
现在又无法获取了，没有登录框，直接给个令牌，不能用，没有关联到账户信息啊

**Reply by Dotbeeleo** [reply]
同样的问题 解决了嘛

**Comment by gohkh0825** [comment]
现在又无法获取了，没有登录框，直接给个令牌，不能用，没有关联到账户信息啊

**Comment by WangMingJunGo** [comment]
An error occurred: API rate limit exceeded for alist-org/docs. Sign in to increase the rate limit

**Comment by shiguanglaoxi** [comment]
https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://alist.nn.ci/tool/baidu/callback&scope=basic,netdisk&qrcode=1

**Reply by Fanshi233** [reply]
这个可以用

**Reply by giorgio-wong** [reply]
避坑，点击链接之前先清理一下上网痕迹，这样就可以扫码登录了。

**Comment by gch0509** [comment]
什么问题发自我的手机-------- 原始邮件 --------发件人： Dotbeeleo ***@***.***>日期： 2023年4月16日周日 下午4:45收件人： alist-org/docs ***@***.***>抄送： gch0509 ***@***.***>, Comment ***@***.***>主    题： Re: [alist-org/docs] tool/baidu/callback (Discussion #146)
同样的问题 解决了嘛

**Comment by endlesswxk** [comment]
OK了

**Comment by JiaVerso** [comment]
为什么上面显示error啊，没办法获取token了吗

**Comment by 99Dabiao** [comment]
failed get objs: failed to list objs: errno: -9, refer to https://pan.baidu.com/union/doc/

**Comment by lamboxr** [comment]
夸克网盘，用tmm削刮器削刮海报，总是会先生成一个大小为0的海报图片或nfo信息，文件名形如“xxxxx-thumb.jpg”然后再生成一个实际有大小的文件，文件后缀加了个(1),甚至（2），（3），形如“xxxxx-thumb（1）.jpg” 导致kodi扫描的时候 不识别大小为0的那个文件名正确的文件

**Comment by NeeYoonc** [comment]
状态: 

Get "https://pan.baidu.com/rest/2.0/xpan/nas?access_token=121.2b5c66e0828a6ae2c713e9c74721a91b.YQNXiaVtHTmCUL8k58fX8X2tGvMi1OH8RiSY8HD.44nPYw&method=uinfo": net/http: TLS handshake timeout

**Reply by xclidongbo** [reply]
failed get objs: failed to list objs: req: [https://pan.baidu.com/rest/2.0/xpan/file] ,errno: -9, refer to https://pan.baidu.com/union/doc/

**Comment by Alan-chang2009** [comment]
我语言点中文为什么会直接跳到主页呀，浏览器自带的翻译又用不了

**Comment by KYYOnull** [comment]
鸡肋项目，没啥卵用

**Comment by slnll** [comment]
获取不了的点这个试试
https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://alist.nn.ci/tool/baidu/callback&scope=basic,netdisk&qrcode=1

**Comment by slycmder** [comment]
我有两个百度网盘，都是VIP，使用第一个网盘挂载后，下载文件的速度很快，但是切换第二个网盘，速度就比较慢了，重新切换回第一个网盘后速度还是很快。我在其他电脑上挂载第二个百度网盘，速度也会很快。怀疑是因为同一个client id使用不同的token导致的，各位有什么方法刷新client id，或者切换百度网盘而不限速的吗？

**Reply by slycmder** [reply]
抱歉，给大家带来困扰了，是我使用限速软件netlimit给alist限速导致的，尚不清楚为什么限速后，切换账号速度会发生变化。

**Comment by JokerWFL** [comment]
这个有问题了？

**Comment by AaronWong625** [comment]
122.88c248aa4a56d380c82b4c05835f1af3.YB1_hnxH5efDxe05TSIQBcouzV-k1s2QukE8mMO.fzqWIw

**Comment by a775566233** [comment]
可以到百度开发者平台上申请

以下对应名称
AppKey: = client_id = 客户端id
SecretKey: = client_secret = 客户端密钥

获取到密钥修改链接 client_id
获取code
openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=填写&redirect_uri=oob&scope=basic,netdisk&qrcode=1

返回code十分钟有效，修改链接
code client_id client_secret
获取token
openapi.baidu.com/oauth/2.0/token?grant_type=authorization_code&code=填写&client_id=填写&client_secret=填写&redirect_uri=oob 

酷安里的教程

**Comment by yunyandegit** [comment]
挂载百度网盘出现了一些问题。
删除出现：errno: 2, refer to https://pan.baidu.com/union/doc/

**Comment by womengnan** [comment]
vip账号，为什么看不了视频和pdf，音频可以，求指点

**Comment by zizhuxiongmao** [comment]
你说的啥，没看懂😒



---Original---
From: ***@***.***&gt;
Date: Tue, Apr 2, 2024 23:50 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [alist-org/docs] tool/baidu/callback (Discussion #146)




 
vip账号，为什么看不了视频和pdf，音频可以，求指点

**Comment by lqysg3** [comment]
https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://alist.nn.ci/tool/baidu/callback&scope=basic,netdisk&force_login=1
用这个可以，用帐密登录就可以

**Comment by wylgis** [comment]
test

**Comment by ganzion** [comment]
Get "https://pan.baidu.com/rest/2.0/xpan/nas?access_token=&method=uinfo": dial tcp: lookup pan.baidu.com on [::1]:53: read udp [::1]:33975->[::1]:53: read: connection refused
提示这个错误，怎么解决

**Comment by 1772891600** [comment]
显示：failed link: failed get link: not find file page param

**Comment by Asexpected** [comment]
20240811反馈不行没有二维码

**Comment by woshichenbenshuai** [comment]
refresh_token 一直转圈呢 提示521错误

**Comment by woshichenbenshuai** [comment]
不行啊一直转圈

**Comment by willbub** [comment]
no response
